docker/kmq/Dockerfile

```diff
@@ -22,6 +22,10 @@ COPY sql_dumps/kmq-test-cached.sql sql_dumps/kmq-test-cached.sql
 COPY sql/ sql/
 COPY templates/ templates/
 
+COPY bin/ bin/
+RUN curl -L https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp -o bin/yt-dlp
```

Comment:
https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#update 

Do we want to use `nightly` or `master` channel instead?